docs/superpowers/plans/2026-05-01-contact-form.md

```
@@ -0,0 +1,1401 @@
+
+**Spec:** `docs/superpowers/specs/2026-05-01-contact-form-design.md`
+
+---
+
+## 事前準備（実装着手前のユーザー作業）
+
+1. Resend API キー発行（https://resend.com/api-keys）
+2. Cloudflare Turnstile サイト追加 → Site Key / Secret Key 発行
+3. プロジェクトルートの `.env` に以下を追記：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)